### PR TITLE
packer: implement waitforclock per-package config files

### PR DESCRIPTION
**Pull request by dtchanpura**
This PR adds a configuration `dontstart` it is related to https://github.com/gokrazy/gokrazy/pull/105.

This allows wait for the clock to be synced before starting the service

packer.go: adds `findWaitForClock()` function similar to `findDontStart()`
buildinit.go: adds an exclusive condition with don't start and start to run ~~`NewDelayedService()`~~ `NewWaitForClockService()`

### CODE REVIEW

File: cmd/gokr-packer/packer.go [hunk 1]
```diff
@@ -404,6 +404,39 @@ func findDontStart() (map[string]bool, error) {
...
+
+	contents := make(map[string]bool)
+	for _, p := range waitForClockPaths {
+		pkg := strings.TrimSuffix(strings.TrimPrefix(p.path, "waitforclock/"), "/waitforclock.txt")
+		if !buildPackages[pkg] {
+			log.Printf("WARNING: waitforclock.txt file %s does not match any specified package (%s)", pkg, flag.Args())
+			continue
+		}
+		packageConfigFiles[pkg] = append(packageConfigFiles[pkg], packageConfigFile{
+			kind:         "wait for the clock before start",
```
Comment (stapelberg) [inline cmd/gokr-packer/packer.go:427]: How about: “wait for clock synchronization before start”?

**Review by stapelberg — COMMENTED** [review]
Looks good overall, but will need an update for the new name of NewDelayedService

File: cmd/gokr-packer/packer.go (hunk 1 above)
Comment (dtchanpura) [reply cmd/gokr-packer/packer.go:427]: Sure, makes sense.

**Review by stapelberg — APPROVED** [review]

**Comment by stapelberg** [comment]
Can you update your PR to resolve conflicts please?

Also, can you pull in the latest `github.com/gokrazy/gokrazy` into go.mod please?

**Comment by dtchanpura** [comment]
Sure, let me do that.

**Comment by stapelberg** [comment]
Thanks!

**Pull request merged**